tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala

```diff
@@ -71,6 +71,7 @@ object SparkSessionHolder extends Logging {
     Locale.setDefault(Locale.US)
     SparkSession.builder()
         .master("local[1]")
+        .config("spark.sql.adaptive.enabled", "false")
```

Comment:
do we have followup for this?

Reply:
I just added this to make it explicit - the default is false already.

Reply:
We run the integration tests both with AQE on and off. We decided not to run the unit tests both ways since it would take so long. I have been manually toggling this config during development though as a sanity check. 

Also, there is a follow-up WIP PR open for porting specific AQE tests over from Spark. This involves further shim work though so I wanted to keep that separate.